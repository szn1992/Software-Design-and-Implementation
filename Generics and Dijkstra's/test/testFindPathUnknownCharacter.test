# Test the correctness of FindPath if there is an unknown character

# Load the graph
LoadGraph marvel marvel.tsv

# Find the path of an unknown node to a known node
FindPath marvel Eric_Sun WISDOM,_PETER

# Find the path of a known node to an unknown node
FindPath marvel WISDOM,_PETER Eric_Sun

# Find the path of an unknown node to an unknown node
FindPath marvel Batman Superman