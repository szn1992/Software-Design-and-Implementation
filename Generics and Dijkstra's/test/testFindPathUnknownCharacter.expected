# Test the correctness of FindPath if there is an unknown character

# Load the graph
loaded graph marvel

# Find the path of an unknown node to a known node
unknown character Eric Sun

# Find the path of a known node to an unknown node
unknown character Eric Sun

# Find the path of an unknown node to an unknown node
unknown character Batman
unknown character Superman